packages/extension/src/components/organisms/BaseTransaction/hooks/useTransactionStatus.tsx

```diff
@@ -10,7 +10,7 @@ import { AsyncTransaction, PageWithSpinner } from '../../../templates';
 interface TransactionStatusProps {
   isParamsMissing: boolean;
   errorDifference: string | undefined;
-  network: Network | undefined;
```

Comment:
I think this file should be moved to `packages/extension/src/hooks` I don't know why it's within this folder.

Reply:
Done